src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -26,25 +26,17 @@
 import org.loklak.objects.AbstractObjectEntry;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Though we are not following those guidelines, I have made the changes. Please take a look.